Copyedit (grammar, spelling, etc.) for clarity

quaid opened the pull request:
- Small grammar, spelling, and punctuation fixes
- A few language shifts or additions to further clarity.
- Refactored to have one-line per sentence for easier reading/reviewing of diffs.

## Related Issues and Dependencies
N/A

## This introduces a breaking change

- [ ] Yes
- [x] No

<!-- If this introduces a breaking change, please describe the impact and migration path for existing applications below. -->

## This Pull Request implements

Small grammar, spelling, and language choice fixes as part of a copyedit.

## Description

<!--- Describe your changes in detail -->
 - Small grammar, spelling, and punctuation fixes
- A few language shifts or additions to further clarity.
- Refactored to have one-line per sentence for easier reading/reviewing of diffs.

quaid committed:
Copyedit (grammar, spelling, etc.) for clarity

- Small grammar, spelling, and punctuation fixes
- A few language shifts or additions to further clarity.
- Refactored to have one-line per sentence for easier reading/reviewing of diffs.

sesheta commented:
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **durandom** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @durandom` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=aicoe-aiops%2Focp-ci-analysis).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/aicoe-aiops/ocp-ci-analysis/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":["durandom"]} -->

sesheta commented:
Hi @quaid. Thanks for your PR.

I'm waiting for a [aicoe-aiops](https://github.com/orgs/aicoe-aiops/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/aicoe-aiops/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=aicoe-aiops%2Focp-ci-analysis).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

oindrillac commented:
/ok-to-test

quaid committed:
Fixing ODH name

Although you may see it spelled with and without spaces, looking at the ODH website it's pretty clear the proper way is "Open Data Hub".

This is important whenever it's a proper name, and more important if it's someone's trademarks. Also, consistency and accuracy help create reader comfort.

oindrillac commented on docs/workshop/prerequisites.md:
```diff
@@ -1,13 +1,17 @@
+You can still complete the workshop without prior Python knowledge.
 
-You do not need to be familiar with these any of tools being used in this workshop, and you should have around an intermediate level of comfort with technology and learning new technical concepts.
+You do not need to be familiar with the any of tools being used in this workshop, and you should have around an intermediate level of comfort with technology and learning new technical concepts.
```
```suggestion
You do not need to be familiar with any of the tools being used in this workshop, and you should have around an intermediate level of comfort with technology and learning new technical concepts.
```

oindrillac commented on docs/workshop/prerequisites.md:
```diff
@@ -1,13 +1,17 @@
+We will be setting aside some time during the workshop to setup Github accounts for attendees who do not have accounts already.
 
-3. **Familiarity with Python** - The AIOps tools has been built using the Python programming language. Familiarity with Python is preferred but not compulsory. One can still complete the workshop without prior Python knowledge.
+3. **Familiarity with Python** - The AIOps tools has been built using the Python programming language.
```
```suggestion
3. **Familiarity with Python** - The AIOps tools have been built using the Python programming language.
```

oindrillac reviewed (changes requested):
Thanks your PR @quaid! 😄
Suggested couple some other minor grammatical changes that I noticed.

quaid committed:
Merge branch 'aicoe-aiops:master' into quaid-prereq-copyedit-1
